medicines/api/src/schema.rs

```diff
@@ -88,3 +109,14 @@ impl SubscriptionRoot {}
 pub fn create_schema() -> Schema {
     Schema::new(QueryRoot {}, MutationRoot {}, SubscriptionRoot {})
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn test_base_64_decode() {
+        let encoded = base64::encode("1229".to_string());
+        assert_eq!(base_64_decode(encoded).unwrap(), 1230);
```

Comment:
Do we really want decode to return a different number than what was encoded? Feel like the 1 should be added elsewhere?

Reply:
yeah maybe the function name should be a bit clearer that this isn't a direct decoding